cla_backend/libs/eligibility_calculator/cfe_civil/property.py

```diff
@@ -0,0 +1,42 @@
+from cla_backend.libs.eligibility_calculator.cfe_civil.conversions import pence_to_pounds
+
+
+def _convert_house(house_data):
+    return {
+        "value": pence_to_pounds(house_data["value"]),
+        "outstanding_mortgage": pence_to_pounds(house_data["mortgage_left"]),
+        "percentage_owned": house_data["share"],
+        "shared_with_housing_assoc": False,
+        "subject_matter_of_dispute": house_data["disputed"],
+    }
+
+
+def _valid_house(house_data):
+    return (
+        "main" in house_data
+        and house_data["value"] is not None
+        and house_data["mortgage_left"] is not None
+        and house_data["share"] is not None
+        and "disputed" in house_data
+    )
+
+
+def translate_property(possible_property_data):
+    property_data = [house for house in possible_property_data if _valid_house(house)]
+    main_homes = [x for x in property_data if x["main"]]
+    non_mains = [x for x in property_data if not x["main"]]
+    if len(main_homes) > 0:
+        main_home_data = main_homes[0]
```

Comment:
Should it be the first home in the list or the most valuable?

Reply:
This code replicates the code in calculator.main_property.py (which picks the first). The domain only has 1 'main' property (CFE Civil's data structures enforce this) so this is bit moot. Currently if 2 main properties are selected in cla-frontend, the backend reports an error and the front-end stalls anyway. (We have created a bug ticket for this edge case)